hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/fees/FeesAndRatesProvider.java

```diff
@@ -121,8 +122,10 @@ private ExchangeRate activeRates() {
     private void readRateSet() throws Throwable {
```

Comment:
instead of throwing Throwable, would be better to throw a more specific exception (e.g. IOException)

Reply:
I will fix it on another PR related to that you are right 100% I just try to aggregate all the issues related to smoke code specificly